docs-src/tut-7-rpc.scrbl

```diff
@@ -183,6 +210,144 @@ invoke our @pyin{main} function.
 
 @(hrule)
 
+Now that we have written an entire @|RPS| game in Python it is time to try
+running it.
+
+First we need to generate TLS certificates.
```

Comment:
:) You're so noble. You really need to have this path second and/or not at all. It should default to the way that doesn't require input from the user. As I mention in another comment, since we're saying these certificates shouldn't be used anyways...  I think it should default to the ones that are hardcoded and be done with it.